Description: Correct pod NAME issue in LedgerSMB/Template/XLS.pm
 Is missing the dash from the pod NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472644&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Template/XLS.pm
+++ b/LedgerSMB/Template/XLS.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::Template::XLS  Template support module for LedgerSMB
+LedgerSMB::Template::XLS - Template support module for LedgerSMB
 
 =head1 SYNOPSIS
 
